src/main/java/org/opentripplanner/routing/impl/DefaultFareServiceFactory.java

```diff
@@ -162,6 +162,9 @@ public static FareServiceFactory fromConfig(JsonNode config) {
         case "dutch":
             retval = new DutchFareServiceFactory();
             break;
+        case "ocra":
```

Comment:
```suggestion
        case "orca":
```

Not https://i.pinimg.com/originals/10/25/6d/10256d26f085a67e4c1f428af68d325d.jpg 😅